### DISCUSSION TITLE
For safety reasons, please remove this item

### DISCUSSION THREAD

**Discussion by hzpurewater** [dani-garcia/vaultwarden#3657] [Ideas]
It is recommended to delete it. 
After checking Remember, the next login will no longer require the input of a verification code. 
Relatively unsafe.
   
![delete it](https://github.com/dani-garcia/vaultwarden/assets/40298933/d00061ad-bbfd-4d33-a6df-f8d45ceeb35b)

**Comment by ontheair81** [comment]
If you consider it unsafe, then just don't use it and everything is fine.

Maybe there are people with different use cases who consider this as a safe-enough variant for their purpose. Especially if they are on computers with encrypted storage and them being the only user with an account on this machine.
If they use the web vault at all, instead of the application or plugin.

If you still consider it unsafe - just because this option exist, without being forced to use it - please explain.

**Comment by hzpurewater** [comment]
I want to use this two-step verification feature, but I don't want this option to be remembered in this place.

**Comment by hzpurewater** [comment]
Once this option is selected, it means that in the future, each login will be reduced by one verification process, which reduces security.

**Comment by ontheair81** [comment]
> I want to use this two-step verification feature

Sure. 2FA is always a good idea.

> but I don't want this option to be remembered in this place.

If you don't want to use an option, don't click on it and you are fine.
That's the nice thing about _options_: They are _optional_, you are not forced to use it.

**Comment by hzpurewater** [comment]
You're right, but when used by multiple people, some people may not pay much attention to this issue. So, I'm asking a question here.Thanks.

**Reply by tessus** [reply]
If you are allowing multiple people accesss to your account on your computer, you have a much bigger problem. Don't even talk about security then.

**Comment by RealOrangeOne** [comment]
This is an issue better raised upstream with Bitwarden than here, as it's an issue isolated to the client, which is identical to the upstream Bitwarden frontend.
